babeld/babel_filter.c

```diff
@@ -62,15 +65,15 @@ babel_filter(int output, const unsigned char *prefix, unsigned short plen,
                    "%pFX filtered by distribute %s",
                    &p, output ? "out" : "in");
             return INFINITY;
-	}
+        }
```

Comment:
Converted tabs to spaces.

Reply:
Please ignore here with styling, babeld is kinda sort of _IMPORTED_ stuff and we keep the formatting as it was (not applying our styling rules).

Reply:
Sure, I've discarded those styling changes.